qmt/data/geometry.py

```diff
@@ -153,24 +155,30 @@ def edge_coord_list(self, edge_name):
 
     def plot(
         self,
-        axis_limits: Sequence[float] = [],
         parts_to_exclude: Sequence[str] = [],
         line_width: float = 20.0,
-        figsize: Sequence[float] = (10, 10),
+        fig: Optional[Figure] = None,
```

Comment:
Here I would put the axis instead too - because subplot is quite restrictive 
See https://stackoverflow.com/questions/23739277/how-should-i-pass-a-matplotlib-object-through-a-function-as-axis-axes-or-figur

Reply:
Done